docker-compose.yaml

```diff
@@ -13,8 +14,9 @@ services:
 #      Warning: this is opening it up to all cross domains
 #      http.cors.allow-origin: "http://localhost"...
       http.cors.allow-origin: "*"
-      http.cors.allow-methods: OPTIONS,HEAD,GET,POST,PUT,DELETE
+      http.cors.allow-methods: OPTIONS,TRACE,HEAD,GET,POST,PUT,DELETE
       http.cors.allow-credentials: true
+      # TODO: add REST headers required here
       http.cors.allow-headers: X-Requested-With,X-Auth-Token,Content-Type,Content-Length,Authorization
```

Comment:
This docker compose is only for building and testing this plugin, so we should only added things that it requires.